app/components/preprint-form-project-select.js

```diff
@@ -184,6 +184,7 @@ export default Component.extend(Analytics, {
                 preprint: false,
                 title: searchTerm,
             },
+            embed: 'parent',
```

Comment:
should `_getMoreUserNodes` also embed parent?

Reply:
Yep! Good catch.

Reply:
Yep! Good catch.